.travis.yml

```diff
@@ -27,7 +29,15 @@ install:
 
 before_script:
   - python setup.py develop
+  - conda activate omnisci-ci
+  - mkdir data && omnisci_initdb data -f 
+  - omnisci_server --enable-runtime-udf --enable-table-functions &
```

Comment:
Suggestion:
```
omnisci_server --enable-runtime-udf --enable-table-functions 2>&1 > omniscidb-output.txt &
```

Reply:
Do we actually need to pipe the output to a file? It seems that travis doesn't support piping inside an `if` clause:
`if [[ some_condition ]]; omnisci_server --enable-runtime-udf --enable-table-functions 2>&1 > omniscidb-output.txt &`

Reply:
see: https://travis-ci.org/github/xnd-project/rbc/builds/705525843

Reply:
The build log shows command
```
if [ "$OMNISCIDB_FROM" = "conda" ]; then omnisci_server --enable-runtime-udf --enable-table-functions &> omniscidb-output.txt & ; fi
```
but it should be
```
if [ "$OMNISCIDB_FROM" = "conda" ]; then omnisci_server --enable-runtime-udf --enable-table-functions 2>&1 omniscidb-output.txt & ; fi
```

Reply:
I think you just had a typo: `&>` ought to be `2>&1`

Reply:
I have tried that as well. See https://travis-ci.org/github/xnd-project/rbc/jobs/705503755

Reply:
Create a `start_omniscidb_bg.sh` script that contains all relevant commands needed to setup and start the omniscidb server.